app/src/main/java/com/connorlay/knitgrid/fragments/PatternDetailFragment.java

```diff
@@ -41,6 +46,8 @@
 
     private PatternPresenter mPatternPresenter;
 
+    private RowColumnListener mListener;
+
     public static PatternDetailFragment newInstance(PatternPresenter patternPresenter) {
```

Comment:
The issue with that is that there is nothing to inject for highlighting. If you look at the way the code works, the listener that is injected is only for communication outside of the fragment. I still have to create an OnClickListener, in parallel with highlighting. For creation, it also calls out to this listener. But that isn't how highlighting works. What were you thinking we could inject?